lib/middleware/authorize.js

```diff
@@ -0,0 +1,11 @@
+module.exports = async (req, res, next) => {
+  try {
+    if (!req.user)
```

Comment:
Currently, any logged in user is able to access the routes protected with this middleware - and we need to limit access for the delete route to users who are admin, or the user who created the review (this will require that you update your insert method to also take in the user_id, so you're able to know which user made each post)